testutils/src/main/scala/za/co/absa/enceladus/testutils/HelperFunctions.scala

```diff
@@ -127,4 +127,17 @@ object HelperFunctions {
     val flatteningFormula: List[Column] = flattenSchema(df)
     df.select(flatteningFormula: _*)
   }
+
+  /**
+    * Calculates times that it took for passed block of code to execute and finish.
+    * @param callback Block of code, to be executed
+    * @tparam A A type parameter specifying callback return
+    * @return Returns a millisecond difference between start and end time
+    */
+  def calculateTime[A](callback: => A): Long = {
```

Comment:
Tiny: Consider different parameter then then `callback`, which could be somewhat misleading.

Reply:
Renamed